awx/ui/src/setupProxy.js

```diff
@@ -1,6 +1,6 @@
 const { createProxyMiddleware } = require('http-proxy-middleware');
 
-const TARGET = process.env.TARGET || 'https://localhost:8043';
+const TARGET = process.env.TARGET || 'https://54.147.146.116:8043';
```

Comment:
I'm not even sure what this is for, exactly, but it seems like it might be a temporary change that you wouldn't want to commit?

Reply:
that's correct.